appveyor.yml

```diff
@@ -21,24 +22,54 @@ matrix:
 
 # skip unsupported combinations
 init:
-- set arch=
-- if "%arch%"=="Win64" ( set arch= Win64)
```

Comment:
Why don't we need `%arch%` anymore. It gets used a few lines down

Reply:
arch is set by appveyor externally. line 25 onle added a whitespace in front of the string which is somehow useless